sql/core/src/test/resources/sql-tests/results/pgSQL/window.sql.out

```diff
@@ -0,0 +1,2526 @@
+NULL	3	3
+NULL	7	3
+
+
+-- !query 90
+SELECT sum(unique1) over (order by four range between unbounded preceding and 2 following),
+unique1, four
+FROM tenk1 WHERE unique1 < 10
+-- !query 90 schema
+struct<sum(unique1) OVER (ORDER BY four ASC NULLS FIRST RANGE BETWEEN UNBOUNDED PRECEDING AND 2 FOLLOWING):bigint,unique1:int,four:int>
+-- !query 90 output
+35	0	0
+35	4	0
+35	8	0
+45	1	1
+45	2	2
+45	3	3
+45	5	1
+45	6	2
+45	7	3
+45	9	1
+
+
+-- !query 91
+SELECT sum(unique1) over (order by four range between 2 preceding and 1 preceding),
+unique1, four
+FROM tenk1 WHERE unique1 < 10
+-- !query 91 schema
+struct<sum(unique1) OVER (ORDER BY four ASC NULLS FIRST RANGE BETWEEN 2 PRECEDING AND 1 PRECEDING):bigint,unique1:int,four:int>
+-- !query 91 output
+12	1	1
+12	5	1
+12	9	1
+23	3	3
+23	7	3
+27	2	2
+27	6	2
+NULL	0	0
+NULL	4	0
+NULL	8	0
+
+
+-- !query 92
+SELECT sum(unique1) over (order by four range between 0 preceding and 0 following),
+unique1, four
+FROM tenk1 WHERE unique1 < 10
+-- !query 92 schema
+struct<sum(unique1) OVER (ORDER BY four ASC NULLS FIRST RANGE BETWEEN 0 PRECEDING AND 0 FOLLOWING):bigint,unique1:int,four:int>
+-- !query 92 output
+10	3	3
+10	7	3
+12	0	0
+12	4	0
+12	8	0
+15	1	1
+15	5	1
+15	9	1
+8	2	2
+8	6	2
+
+
+-- !query 93
+SELECT sum(unique1) over (partition by ten
+  order by four range between 0 preceding and 0 following),unique1, four, ten
+FROM tenk1 WHERE unique1 < 10
+-- !query 93 schema
+struct<sum(unique1) OVER (PARTITION BY ten ORDER BY four ASC NULLS FIRST RANGE BETWEEN 0 PRECEDING AND 0 FOLLOWING):bigint,unique1:int,four:int,ten:int>
+-- !query 93 output
+0	0	0	0
+1	1	1	1
+2	2	2	2
+3	3	3	3
+4	4	0	4
+5	5	1	5
+6	6	2	6
+7	7	3	7
+8	8	0	8
+9	9	1	9
+
+
+-- !query 94
+WITH cte (x) AS (
+        SELECT * FROM range(1, 35, 2)
+)
+SELECT x, (sum(x) over w)
+FROM cte
+WINDOW w AS (ORDER BY x rows between 1 preceding and 1 following)
+-- !query 94 schema
+struct<x:bigint,sum(x) OVER (ORDER BY x ASC NULLS FIRST ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING):bigint>
+-- !query 94 output
+1	4
+11	33
+13	39
+15	45
+17	51
+19	57
+21	63
+23	69
+25	75
+27	81
+29	87
+3	9
+31	93
+33	64
+5	15
+7	21
+9	27
+
+
+-- !query 95
+WITH cte (x) AS (
+        SELECT * FROM range(1, 35, 2)
+)
+SELECT x, (sum(x) over w)
+FROM cte
+WINDOW w AS (ORDER BY x range between 1 preceding and 1 following)
+-- !query 95 schema
+struct<x:bigint,sum(x) OVER (ORDER BY x ASC NULLS FIRST RANGE BETWEEN CAST((- 1) AS BIGINT) FOLLOWING AND CAST(1 AS BIGINT) FOLLOWING):bigint>
+-- !query 95 output
+1	1
+11	11
+13	13
+15	15
+17	17
+19	19
+21	21
+23	23
+25	25
+27	27
+29	29
+3	3
+31	31
+33	33
+5	5
+7	7
+9	9
+
+
+-- !query 96
+WITH cte (x) AS (
+        SELECT * FROM range(1, 35, 2)
+)
+SELECT x, (sum(x) over w)
+FROM cte
+WINDOW w AS (ORDER BY x range between 1 preceding and 1 following)
+-- !query 96 schema
+struct<x:bigint,sum(x) OVER (ORDER BY x ASC NULLS FIRST RANGE BETWEEN CAST((- 1) AS BIGINT) FOLLOWING AND CAST(1 AS BIGINT) FOLLOWING):bigint>
+-- !query 96 output
+1	1
```

Comment:
different output from pg: https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/expected/window.out#L2700

Reply:
ditto: the problem is the fact that I changed `groups` to `range` because I thought that they had the same behavior, which is not the case. I already reverted it.